libenv/sysinfo.c

```diff
@@ -1430,6 +1464,11 @@ static void OSClasses(EvalContext *ctx)
 
 #ifdef _AIX
     strlcpy(vbuff, VSYSNAME.version, CF_MAXVARSIZE);
+
+    char strminor[CF_MAXVARSIZE];
+    snprintf(strminor, CF_MAXVARSIZE, "%s", VSYSNAME.release);
```

Comment:
Would be nice to add an assert to test the assumption that the buffer is large enough.
```suggestion
    NDEBUG_UNUSED int ret = snprintf(strminor, CF_MAXVARSIZE, "%s", VSYSNAME.release);
    assert(ret >= 0 && (size_t) ret < sizeof(strminor));
```